src/blocks.rs

```diff
@@ -19,6 +19,20 @@ pub async fn descriptions(client: &Client, cursor: Option<&str>) -> Result<Descr
         .map(|Data { data, cursor }| Descriptions { data, cursor })
 }
 
+pub fn transactions_at_height(client: &Client, block: u64) -> Stream<Transaction> {
+    client.fetch_stream(
+        format!("/blocks/{}/transactions", block.to_string()).as_str(),
+        NO_QUERY,
+    )
+}
+
+pub fn transactions_at_hash(client: &Client, hash: &str) -> Stream<Transaction> {
```

Comment:
`transaction_with_hash` would read better to me

Reply:
Should it be `transaction_with_hash` or `transactions_with_hash`?

Reply:
That is not the transaction hash. It is the hash of the block. It implements the GET https://api.helium.io/v1/blocks/hash/:hash/transactions from https://docs.helium.com/api/blockchain/blocks. Maybe we should rename hash var to block_hash to make it more evident?

Reply:
Ah, woops! My bad.

You're 100% right. This all looks good to me the way it was initially. "at" hash was already right.

Reply:
rename the _function_ to `transactions_at_block_hash`  to not hide it in the parameter name?

Reply:
Done.